pkg/registry/core/service/allocator/bitmap.go

```diff
@@ -124,6 +124,34 @@ func (r *AllocationBitmap) Release(offset int) error {
 	return nil
 }
 
+const (
+	// Find the size of a big.Word in bytes.
+	notZero   = ^big.Word(0)
+	wordPower = notZero>>8&1 + notZero>>16&1 + notZero>>32&1
+	wordSize  = 1 << wordPower
+	notOne    = ^big.Word(1)
+)
+
+// ForEach calls the provided function for each allocated bit.  The
+// AllocationBitmap may not be modified while this loop is running.
+func (r *AllocationBitmap) ForEach(fn func(int)) {
```

Comment:
nit: I vaguely recall us using Each() elsewhere - if so would be good to be consistent.  If not, I don't care.